flink-core/src/test/java/org/apache/flink/api/common/serialization/SimpleStringSchemaTest.java

```diff
@@ -50,4 +50,12 @@ public void testSerializability() throws Exception {
 
 		assertEquals(schema.getCharset(), copy.getCharset());
 	}
+
+	@Test
+	public void testDeserializeWithNullValue() throws Exception {
+		final SimpleStringSchema schema = new SimpleStringSchema();
+		final String value = schema.deserialize(null);
+
+		assertEquals(value, null);
```

Comment:
Expected value is always the first argument in junit. Also consider using `assertNull`.

Reply:
right